src/SFA.DAS.AssessorService.Application.Api/Services/StandardService.cs

```diff
@@ -76,6 +76,22 @@ public async Task<StandardCollation> GetStandard(string referenceNumber)
             return standardCollation;
         }
 
+        public async Task<List<Option>> GetOptions(int stdCode)
+        {
+            List<Option> options = null;
+
+            try
+            {
+                options = await _standardRepository.GetOptions(stdCode);
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex, $"OPTION: Failed to get for options for stdcode: {stdCode}");
```

Comment:
Should the error be propagated rather than swallowed here?

Reply:
Remove the whole method as it could be refactored away